File: escalon.m
% Ian Mu;oz Nu;ez - Funcion escalon o umbral

close all
clear
clc

v = linspace(-10, 10, 1000);

phi = [];
for i=1:1000
    if v(i) >= 0
        phi(i) = 1;
    else
        phi(i) = 0;
    end
end

figure(1)
hold on
grid on

plot(v, phi, 'g', 'LineWidth', 2)

title("Funcion escalon o umbral", 'FontSize', 20)
xlabel('v', 'FontSize', 15)
ylabel('\phi(v)', 'FontSize', 15)

File: escalon.py
# Ian Mu;oz Nu;ez - Funcion escalon o umbral

import matplotlib.pyplot as plt
import numpy as np

n = 1000

v = np.linspace(-10, 10, n)

phi = np.zeros(n)
for i in range(0, n):
    if v[i] >= 0:
        phi[i] = 1
    else:
        phi[i] = 0

plt.figure(1)
plt.grid()

plt.plot(v, phi, 'y-', linewidth=2)

plt.title("Funcion escalon o umbral", fontsize=20)
plt.xlabel('v', fontsize=15)
plt.ylabel(r'$\phi$(v)', fontsize=15)

plt.show()

